src/components/panels/ToolheadControlPanel.vue

```diff
@@ -94,9 +94,10 @@
         <!-- MOVE TO CONTROL -->
         <move-to-control class="py-0 pt-3"></move-to-control>
         <!-- AXIS CONTROL -->
-        <v-container>
+        <v-container v-if="axisControlVisible">
             <component :is="`${controlStyle}-control`"></component>
         </v-container>
+        <v-container v-if="!axisControlVisible" class="py-0 pt-3"></v-container>
```

Comment:
pls remove this line and add a margin-top to the next v-divider when you hide the control area.

Reply:
done, thanks for the hint. It's better this way 👍